src/main/java/myjournal/Ui.java

```diff
@@ -0,0 +1,80 @@
+package myjournal;
+
+import myjournal.task.Task;
+
+/**
+ * Creates the Ui object.
+ *
+ * @author felissafaustine
+ */
+public class Ui {
+    /**
+     * Prints the welcome message.
+     * @return
+     */
+    public static String welcomeMessage() {
+        return "Hello!\n"
+                + "1. Type a task (todo/event/deadline) to be added into your task list,\n"
+                + "followed by the time for events and due date for deadlines \n"
+                + "in form of yyyy-mm-dd for date and hh:mm for time.\n"
+                + "2. Type 'list' if you want to generate your task list.\n"
+                + "3. Type 'done [number]' to mark a task as done.\n"
+                + "4. Type 'delete [number]' to delete a task.\n"
+                + "5. Type 'find [keyword]' to get the list of tasks with that keyword.\n"
+                + "6. Type 'bye' to exit";
+    }
+
+    /**
+     * Prints out the statement after a task is added.
+     *
+     * @param taskList The list of all tasks.
+     */
+    public String taskAddPrint(TaskList taskList) {
+        return "Okay!! I've added the following task:\n"
+                + taskList.getTask(taskList.getSize() - 1) + "\n"
+                + "Now you have " + taskList.getSize() + " in the list";
+    }
```

Comment:
I realised you seem to forget to add `@return` for most of your JavaDoc for methods that return something in this Ui class?